lib/asciidoctor/js/asciidoctor_ext/node/abstract_node.rb

```diff
@@ -6,7 +6,7 @@ class AbstractNode
   def generate_data_uri_from_uri image_uri, cache_uri = false
     %x{
       var xhr = new __XMLHttpRequest__();
-      xhr.open('GET', uri, false);
+      xhr.open('GET', image_uri, false);
```

Comment:
The good thing about running a compiler is that it even found a bug!

Reply:
Test added and bug fixed in https://github.com/asciidoctor/asciidoctor.js/pull/662

Reply:
Nice!